src/AddTodoForm.js

```diff
@@ -1,26 +1,24 @@
 import React, { useState } from 'react';
 
+
 const AddTodoForm = ({onAddTodo}) => {
 
   const [todoTitle,setTodoTitle]= useState('');
 
   const handleTitleChange  =(event) =>{
-       
-        setTodoTitle(event.target.value);
+      const newTodoTitle = event.target.value;
+       setTodoTitle(newTodoTitle);
   }
 
  
  const handleAddTodo = (event) =>{
       event.preventDefault();
+      const todoObj = {title:todoTitle, id:Date.now()}
+
 
-     
-      const newTodo = {
-        title:todoTitle,
-        id: Date.now(),
-      };
-      console.log(newTodo)
+      console.log(todoObj )
 
-      onAddTodo(newTodo);
+      onAddTodo(todoObj);
```

Comment:
You could reduce these line into a single line. 

```suggestion
      onAddTodo({title: todoTitle, id: Date.now()});
```